src/Nest/Mapping/Types/Specialized/Attachment/AttachmentAttribute.cs

```diff
@@ -1,5 +1,8 @@
-﻿namespace Nest
+﻿using System;
+
+namespace Nest
 {
+	[Obsolete("Removed in 6.0.")]
```

Comment:
I think we can help users here by pointing them to using the `ingest-attachment` plugin in the comment.

Reply:
:+1: